ff/src/biginteger/mod.rs

```diff
@@ -549,20 +549,9 @@ impl<const N: usize> BigInteger for BigInt<N> {
 
     #[inline]
     fn from_bits_be(bits: &[bool]) -> Self {
-        let mut res = Self::default();
-        let mut acc: u64 = 0;
-
         let mut bits = bits.to_vec();
         bits.reverse();
-        for (i, bits64) in bits.chunks(64).enumerate() {
-            for bit in bits64.iter().rev() {
-                acc <<= 1;
-                acc += *bit as u64;
-            }
-            res.0[i] = acc;
-            acc = 0;
-        }
-        res
+        Self::from_bits_le(&bits)
```

Comment:
Could you please add a test to confirm that the logic is the same?

Reply:
Done, just added two unit tests in the suite in order to confirm the logic in both directions.

Reply:
Great, thanks!